trainee_template/analysis.amlg

```diff
@@ -509,7 +509,10 @@
 								(not (contains_index inactiveFeaturesMap (current_value)))
 								(not (contains_index uniqueNominalsSet (current_value)))
 								(>
-									(- num_cases (get featureNullRatiosMap (list (current_value 1) "num_nulls")) )
+									(-
+										num_cases
```

Comment:
rather than (- num_cases num_nulls) to get the amount of non-null cases, could we just do a (query_not_equals)?